### ISSUE TITLE
Left corner icon

### ISSUE THREAD

**Issue by Jesterjke** [kirigayakazushin/deepin-topbar#24]
1. I saw some youtube videos, where when you switch to another app icon in the left corner changes accordingly.
For me it doesn't work, it is only deepin logo all the time.
2. Also when you click on this icon to open dropdown menu and after that click somewhere outisde topbar it doesnt lose focus and hides - it stays open.
3. What should item "Preference setting" in this dropdown menu do? For me it does nothing when I click it.

**Comment by kirigayakazushin** [comment]
@Jesterjke  I made some changes and fixed on the upper left corner of the display of deepin logo.about second point, this is a bug of deepin, it will be repaired after beta. "Preference setting" is still supported, I plan to write a full settings window  to replace the control center. It may be a long time.

**Comment by Jesterjke** [comment]
@kirigayakazushin, I don't see any new commits in dev&master branches, maybe you forgot to push changes?
About "Preference setting". I do not understand you. What should it do now? For now if click on it, nothing happens. Should it open control center now?

**Comment by kirigayakazushin** [comment]
@Jesterjke  Preference setting will open dde-control-center.  version 0.3.3.1-1 updated.

**Comment by Jesterjke** [comment]
@kirigayakazushin still no changes. Deepin logo in left upper corner persists all the way, no matter which app I am opening.
![image](https://user-images.githubusercontent.com/22295967/32978033-a77b1a18-cc4a-11e7-97e2-d1a95367b3e8.png)
![image](https://user-images.githubusercontent.com/22295967/32978036-b7e54770-cc4a-11e7-9326-81fd5ddf4dfc.png)

**Comment by kirigayakazushin** [comment]
@Jesterjke My point is to keep the logo as deepin.

**Comment by Jesterjke** [comment]
@kirigayakazushin Okay, got you. But I think it would be more cool to change icon, when you switch the applications. Or you can do this as option.

**Comment by andryNoor** [comment]
I agree with you @Jesterjke it will be more cool if we have that feature

**Comment by dimitrov-adrian** [comment]
Actually it doesn't matter a lot what icon it show, if the current app name is noticeable as it's now.

I have just two suggestions

1. Make the logo icon black white like in the macOS
2. Decrease the giant font-size and may be make it bold

Also the language selector is not very visible as it's light text on light background, when tried to change theme to dark, then the texts are not readable.

Actually I like how macOS looks on my computer

<img width="1437" alt="screen shot 2017-12-13 at 9 14 02" src="https://user-images.githubusercontent.com/1009639/33926311-4229a7ce-dfe6-11e7-98af-88758f8920e2.png">

May be if you fix the dark theme will be better.

**Comment by kirigayakazushin** [comment]
@dimitrov-adrian  thank you for you picture. I'm preparing a black-and-white theme, and I need a lot of adjustment.
